### ISSUE TITLE
Update sidebar news heading level

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#2712]
## Problem/Motivation
I want to meet accessibility standards

### Describe the bug

Sidebar news headings are H6 instead of H3

## Proposed resolution
Description of the proposed solution, the rationale behind it, and workarounds for people who cannot use the patch.

##  Screenshots
If applicable, add screenshots to help explain your problem. (Need a tool to share screenshots or screencasts as URLs? Try [monosnap](https://monosnap.com)
